packages/wp-source/src/actions.ts

```diff
@@ -122,6 +122,14 @@ const actions: WpSource["actions"]["source"] = {
 
     libraries.source.api.init({ api, isWpCom });
 
+    // If the URL contains an auth token, then add it to the state.
+    // This is normally the case e.g, when accessing the post preview.
+    const auth = state.frontity?.options?.frontitySourceAuth;
```

Comment:
The `frontity` prefix is removed from query options, isn't it?

Shouldn't it be `sourceAuth` instead `frontitySourceAuth`?

Reply:
Solved :+1: